METAPROGRAMMING.md

```diff
@@ -77,10 +86,187 @@ function observed({kind, key, placement, descriptor, initializer}, get, set) {
         window.requestAnimationFrame(this.render);
       },
       enumerable: descriptor.enumerable,
-      configurable: descriptor.configurable,
+      configurable: descriptor.configurable
     },
     extras: [underlying]
   };
-  
 }
 ```
+
+<a id="bound-decorator-notes"></a>
+## Notes about the `@bound` decorator
```

Comment:
This whole section is a bit different from the rest of the document. The above document is really supposed to be an introduction to decorators. `@bound` is just an example that shows how things work. I don't want to take a strong stance here, as it's a bit of a distraction; decorators seem well-motivated even if we don't have `@bound`. Could you move this text to another location (could be another document inside or outside of this repository).